server/services/twitterStrategy.js

```diff
@@ -0,0 +1,47 @@
+import passport from 'passport';
+import {
+  Strategy as TwitterStrategy
+} from 'passport-twitter';
+import dotenv from 'dotenv';
+import passportAuthenticator from '../utilities/passportAuthentication';
+import Model from '../models';
+import generateToken from '../utilities/jwtGenerator';
+
+const { User } = Model;
+
+dotenv.config();
+passport.serializeUser((user, done) => {
```

Comment:
whats this for?

Reply:
Yeah, It's for the express-session twitter's oauth1.0 is enforcing. It works with the session to serialise and deserialise user data.